wcfsetup/install/lang/de.xml

```diff
@@ -1505,6 +1505,8 @@ Als Benachrichtigungs-URL in der Konfiguration der sofortigen Zahlungsbestätigu
 		<item name="wcf.acp.option.message_sidebar_enable_articles"><![CDATA[Artikelanzahl der Autoren anzeigen]]></item>
 		<item name="wcf.acp.option.log_missing_language_items"><![CDATA[Fehlende Sprachvariablen protokollieren]]></item>
 		<item name="wcf.acp.option.log_missing_language_items.description"><![CDATA[Die fehlenden Sprachvariablen werden in der Datei <code>{'WCF_DIR'|constant}log/missingLanguageItems.txt</code> protokolliert.]]></item>
+		<item name="wcf.acp.option.enable_deprecation_warnings"><![CDATA[Fehler bei Verwendung von @deprecated code]]></item>
```

Comment:
Put `@deprecated` in a `kbd` element? (Same question for the English version and the description.)